grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -269,7 +269,7 @@ class UrlMappings {
         }
 
         "/api/stockMovements/importPickListItems/$id"(parseRequest: true) {
-            controller = "stockMovementApi"
+            controller = "picklist"
```

Comment:
Eventually, if we did want to expose this functionality (i.e. data import endpoint) via the API we could do it in a few different ways

1. Send (POST, PUT, PATCH?) the raw file to the items endpoint
    ```
    PUT /api/picklists/:id/items
    ContentType: multipart/form-data
    
     ```
1. Send CSV as request body which requests the server to handle different input formats (JSON, CSV, XML)
    ```
    PUT /api/picklists/:id/items
    ContentType: text/csv
    Body: 
    id,col1,col2,col3,col4
    a1,b1,c1,d1,e1
    a2,b2,c2,d2,e2
    ```
1. Client-side converts the CSV to JSON 
    ```
    PUT /api/picklists/:id/items
    ContentType: application/json
    Body: 
    [{ ... }, { ... }, { ... }]
    ```

Much later we can potentially talk about a standalone data imports endpoint. 
1. Create a data imports resource to take all of the information required to handle the data. I don't want to spend much time talking about this, but essentially you'll need the request to be a multipart request or contain a body with CSV or JSON, along with any metadata about the import. The request just initiates the data import process. You would use the URL in the Location header to poll for OR if we ended up integrating web sockets or using a messaging system, we could have the client be told when updates are ready and then it would do a GET of the import status, including validation errors, etc. THe beautfy of this is that the client could be anything/anywhere i.e. a background job, a React user interface, a mobile application, an integration medidator, etc.
    Request
    ```
    POST /openboxes/api/data-imports/
    ```
    Response
    ```
    202 Accept 
    Location: /openboxes/api/imports/:uniqueId/status
    ```